src/components/legend/style.js

```diff
@@ -172,13 +172,14 @@ function styleBars(d) {
     barpath.exit().remove();
     barpath.each(function(d) {
         var w = (d.mlw + 1 || markerLine.width + 1) - 1,
```

Comment:
Done in https://github.com/plotly/plotly.js/pull/1289/commits/81d3db1815055e83c9fa3411042fe38caeb5651c